raiden-ts/src/transport/epics/webrtc.ts

```diff
@@ -164,7 +164,7 @@ function matrixWebrtcEvents$<T extends RtcEventType>(
 
 type RtcConnPair = readonly [RTCPeerConnection, RTCDataChannel];
 function isRtcConnPair(value: unknown): value is RtcConnPair {
-  return Array.isArray(value) && value.length === 2 && value[0] instanceof RTCPeerConnection;
```

Comment:
Just out of curiosity (maybe you mentioned it Friday, but then I forgot). Why is it better to avoid `instanceof`?

Reply:
JS uses prototype-based inheritance, which will identity-check if the constructor of the class passed as param is in the prototype chain of the object being tested. This gives some trouble with `instanceof` when you have multiple deps version in `node_modules` on a mono-repo structure, where `instanceof` will fail if the object was created with a constructor on a different dependency folder.
Also, on this specific case, it was changed because I was mocking the `RTCPeerConnection` using a plain object, and then the `instanceof` failed.

Reply:
Thanks, that makes sense.